test/test_files/issue/issue2.test

```diff
@@ -123,3 +123,8 @@
 -STATEMENT MATCH  (f) - [r:etable] -> (t) WHERE  t.id = 'ewm8ewnn@l23:' and r.eprop3 = 'qqewj932-ew2b-5f88-nfew' RETURN COUNT(*);
 ---- 1
 1
+
+-CASE 3416
```

Comment:
i see duckdb allows user to import/export an empty database.
Maybe we should allow as well?